### PR TITLE
Refactor doctor capacity page to use Tailwind

### LINKED ISSUE #4023: Replace Download Live Patients and All Patients to DropdownMenu as Export
**Is your feature request related to a problem? Please describe.**
As discussed in EOD, Download all patients and the live list can be shrunk to one menu button.

**Describe the solution you'd like**
Replace Download Live Patients and All Patients to DropdownMenu as Export

### LINKED ISSUE #4203: Upgrade the facility doctor capacity to tailwind
image

- [ ] Replace the dropdown with the new tailwind dropdown
- [ ] Replace the material UI buttons with tailwind

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #4203

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

**Comment by nihal467** [comment]
@Ashesh3 
image

make the three button in three rows when it is in responsive view

**Comment by Ashesh3** [comment]
> @Ashesh3 image
> 
> make the three button in three rows when it is in responsive view

Done!
image

**Comment by nihal467** [comment]
LGTM

File: src/Components/Facility/DoctorCapacityForm.tsx [hunk 1]
```diff
@@ -232,99 +226,83 @@ export const DoctorCapacityForm = (props: DoctorCapacityProps) => {
...
+            <div className="p-4">
+              <div className="flex justify-between flex-col md:flex-row">
+                <div className="flex flex-row w-full sm:w-auto gap-4">
+                  <ButtonV2
                     id="doctor-cancel"
+                    variant="secondary"
                     className="w-full sm:w-auto"
-                    color="default"
-                    variant="contained"
                     onClick={() => goBack(!id && `/facility/${facilityId}`)}
```
Comment (khavinshankar) [inline src/Components/Facility/DoctorCapacityForm.tsx:276]: As these buttons are inside a form tag, clicking on them would submit the form by default, so use event.preventDefault() or type="button"

```suggestion
                    onClick={(e) => {
                        e.preventDefault()
                        goBack(!id && `/facility/${facilityId}`);
                    }}
```

**Review by khavinshankar — CHANGES_REQUESTED** [review]

**Comment by Ashesh3** [comment]
@khavinshankar Fixed now!

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**